<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KTESnippetRepository</class>
 <widget class="QWidget" name="KTESnippetRepository">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>409</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Snippet Repository:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="lstSnippetFiles">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>280</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KPushButton" name="btnNew">
     <property name="text">
      <string>New Snippet File...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KPushButton" name="btnGHNS">
     <property name="text">
      <string>Get New Snippets...</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="KUrlRequester" name="urlSource"/>
     </item>
     <item>
      <widget class="KPushButton" name="btnCopy">
       <property name="text">
        <string>Copy to repository</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
